source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.rst

```diff
@@ -95,15 +95,9 @@ Default Connection String Options
 As a second preprocessing step, a Client MUST also query the DNS server for
 TXT records on the ``{hostname}``. If available, these
 TXT records provide default connection string options. In most cases only one
-TXT record is necessary. The maximum length of a TXT record string is 255
-characters, but there can be multiple strings per TXT record. A Client MUST
-support multiple TXT record strings and concatenate them as if they were one
-single string in the order they are defined in each TXT record. The order of
-multiple character strings in each TXT record is guaranteed. The maximum
-total length of all strings in a single TXT record is about 65535 characters,
-but DNS providers and software might arbitrarily limit this to a smaller
-number. To allow for this case, a Client MUST allow for multiple TXT records
-for the same host name and consider the options in all of them.
+TXT record is necessary. Because the maximum length of a TXT record string is
+255 characters, a Client MUST allow for multiple TXT records for the same host
+name and consider the options in all of them.
```

Comment:
Let's say here that they're joined with "&". I know you say it below but it's only in the example, let's also state it here in the instructions.